src/styles/header.scss

```diff
@@ -0,0 +1,63 @@
+@include header-max-width(1024px);
+@include header-max-width(1200px);
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+
+  box-shadow: 1px 2px 4px 1px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    &-img {
+      display: block;
+      margin: 10px 0;
+    }
+  }
+
+  &__nav {
+    &-item {
+      margin-left: 20px;
+      position: relative;
+
+      &:first-child {
+        margin-left: 0;
+      }
```

Comment:
except of set on 21 and reset on 24-26 lines it is better to use `:not` `&:not(:first-child) {margin-left: 20px}`